src/bttrx_wifi.cpp

```diff
@@ -164,9 +166,21 @@ void BTTRX_WIFI::setup(BTTRX_CONTROL *control)
 	}
 	bttrx_control_ = control;
 
+	Serial.println("");
 	Serial.println("Configuring access point...");
 
-	WiFi.softAP(WIFI_SSID, WIFI_PASSWORD);
+	String callsign = preferences.getString("callsign", "");
+	if (callsign != "") {
+		char result[sizeof(WIFI_SSID)+7];
```

Comment:
Please do not use magic numbers. Define the maximum length of the callsign in settings.h

Reply:
Done with 0d3d0a5ebce5aeddeaefd9afba42c2d5f3ce572b